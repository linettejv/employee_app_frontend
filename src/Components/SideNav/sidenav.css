.div1 {
    background-color: rgb(240, 239, 238);
    align-items: center;
}

.top-logo {
    padding: 10px 40px 10px 40px;
    align-self: center;
    width: 100%;
    height: 15%;
}

.sidebar {
    height: 100vh;
    display: flex;
    justify-content: center;
    /* align-items: center; */
    background-color: white;
}

aside {
    height: 100vh;
    width: 15%;
    float: left;
}

* {
    margin: 0;
    padding: 0;
    box-sizing: border-box;
}

.icon-em{
    margin-left: 20px;
    width: 30px;
    height: 30px;
    background-color: white;
    border-radius: 50%;
    padding-left: 5px;
    padding-right: 5px;
  }

  .sidebar1{
    padding-top: 15px;
    margin-top: 20px;
    margin-left: 7px;
    display: flex;
    background-color: #04b0f5;
    height: 60px;
    width: 100%;
    border-top-left-radius: 30px;
    border-bottom-left-radius:30px ;
  }

  .emp-list{
    margin-left: 10px;
    padding-top: 5px;
    font-family: 'Montserrat';
    font-weight: 300;
    color: white;
    font-weight: bold;
    
  }
