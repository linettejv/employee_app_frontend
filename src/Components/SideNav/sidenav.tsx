import React from 'react';
import './sidenav.css';

type SideNavPropsTypes = {};

const SideNav: React.FC<SideNavPropsTypes> = () => {
  return (
    <aside>
      <div className='div1'>
        <img className='top-logo' src='/assets/img/kv-logo.png' alt='logo' />
      </div>
      <div className='sidebar'>
        <div className='sidebar1'>
          <img className='icon-em' src='/assets/img/icon.svg' alt='icon-em' />
          <p className='emp-list'>Employee List</p>
        </div>
      </div>
    </aside>
  );
};

export default SideNav;
