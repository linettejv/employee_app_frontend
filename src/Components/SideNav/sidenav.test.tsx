import { render } from '@testing-library/react';
import SideNav from './sidenav';

describe('Testing SideNav', () => {
  test('Checking snapShots:', () => {
    const element = render(<SideNav />);

    expect(element).toMatchSnapshot();
  });
});
