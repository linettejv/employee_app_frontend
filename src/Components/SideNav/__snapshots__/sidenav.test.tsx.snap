// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing SideNav Checking snapShots: 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <aside>
        <div
          class="div1"
        >
          <img
            alt="logo"
            class="top-logo"
            src="/assets/img/kv-logo.png"
          />
        </div>
        <div
          class="sidebar"
        >
          <div
            class="sidebar1"
          >
            <img
              alt="icon-em"
              class="icon-em"
              src="/assets/img/icon.svg"
            />
            <p
              class="emp-list"
            >
              Employee List
            </p>
          </div>
        </div>
      </aside>
    </div>
  </body>,
  "container": <div>
    <aside>
      <div
        class="div1"
      >
        <img
          alt="logo"
          class="top-logo"
          src="/assets/img/kv-logo.png"
        />
      </div>
      <div
        class="sidebar"
      >
        <div
          class="sidebar1"
        >
          <img
            alt="icon-em"
            class="icon-em"
            src="/assets/img/icon.svg"
          />
          <p
            class="emp-list"
          >
            Employee List
          </p>
        </div>
      </div>
    </aside>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
